src/ddscxx/tests/CMakeLists.txt

```diff
@@ -16,6 +16,14 @@ idlcxx_generate(TARGET ddscxx_test_types FILES data/Space.idl data/HelloWorldDat
 configure_file(
   config_simple.xml.in config_simple.xml @ONLY)
 
+if (DDS_HAS_SHM AND NOT WIN32)
+  find_package(iceoryx_posh QUIET)
+  find_package(iceoryx_posh_roudi_environment QUIET)
```

Comment:
We need this to have testing fixture to start and stop the RouDi for every test case